scheduler/src/cook/quota.clj

```diff
@@ -142,22 +143,70 @@
                           count-field
                           (conj [:db/retract [:quota/user user] :quota/count count-field]))))))
 
+(defn- pool-name->type->user->quota
```

Comment:
@scrosby I didn't see much performance benefit from the aggregating quota types change and also didn;t like how that that code ended up. So leaving the resource type aggregation out.

Reply:
Sounds good. We don't want to make this code harder to understand for only a little benefit.